### PR TITLE
Add py versions testing for Zuul

### PR DESCRIPTION
**Pull request by noonedeadpunk**
❓ **What kind of change does this PR introduce?**
  - [ ] 🐞 bug fix
  - [ ] 🐣 feature
  - [ ] 📋 docs update
  - [x] 📋 tests/coverage improvement
  - [ ] 📋 refactoring
  - [ ] 💥 other

📋 **Other information**:

This PR adds cheroot zuul checks using py27, py35, py36, py37 and py38

📋 **Checklist**:

  - [ ] I think the code is well written
  - [ ] I wrote [good commit messages][1]
  - [ ] I have [squashed related commits together][2] after the changes have been approved
  - [ ] Unit tests for the changes exist
  - [ ] Integration tests for the changes exist (if applicable)
  - [ ] I used the same coding conventions as the rest of the project
  - [ ] The new code doesn't generate linter offenses
  - [ ] Documentation reflects the changes
  - [ ] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

---
This change is [Reviewable](https://reviewable.io/reviews/cherrypy/cheroot/269)

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -57,7 +80,9 @@
     check:
       jobs:
       - tox-pre-commit
-      - tox-setup-check
+      - tox-py36-setup-check
```
Comment (webknjaz) [inline .zuul.yaml]: How about 3.8?

**Comment by mnaser** [comment]
recheck

File: .zuul.yaml (hunk 1 above)
Comment (noonedeadpunk) [reply .zuul.yaml]: done

**Comment by mnaser** [comment]
@webknjaz this should be ready and good to go :)

we have some optimizations coming up to avoid building from source with pyenv every time (instead using pre-built python).

**Comment by webknjaz** [comment]
recheck

**Comment by webknjaz** [comment]
@mnaser thanks!
@noonedeadpunk дякую!

**Pull request merged**